packages/utilities/src/EventGroup.ts

```diff
@@ -200,7 +200,7 @@ export class EventGroup {
         eventName: eventName,
         parent: parent,
         callback: callback,
-        useCapture: useCapture || false
+        useCapture: options !== undefined && typeof options === 'boolean'
```

Comment:
Take a look at `removeEventListener` down below in this file. It ends up passing in useCapture from the record.

I think you should change this useCapture to just `options` and cache options. The purpose of having this in the record is so that we pass the same thing back into `removeEventListener`.